src/main/java/Duke/commands/Command.java

```diff
@@ -0,0 +1,26 @@
+package Duke.commands;
+
+import Duke.storage.Storage;
+import Duke.tasks.TaskList;
+import Duke.ui.UI;
+
+import java.time.format.DateTimeFormatter;
+
+public abstract class Command {
```

Comment:
I like the use of an abstract class here, as well as your great organization in the `commands` package!